samples/universal-react-node/package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "bluebird": "^3.4.6",
     "electrode-archetype-react-app": "../../packages/electrode-archetype-react-app",
+    "electrode-archetype-opt-react": "../../packages/electrode-archetype-opt-react",
```

Comment:
is `electrode-archetype-react-app` not working w/o this?

Reply:
whenever `react-notify-toast` is present as a dependency it uses react `15.x`

Reply:
I guess it happens because `electrode-archetype-react-app` does not have `react` package as a direct dependency, but `react-notify-toast` does. Tested with `yarn` (and `npm`), not with `fyn`

Reply:
Because we use `optionalDependencies` `yarn` doesn't work well.

yeah, the issue with react-notify-toast is known.  There's a PR https://github.com/jesusoterogomez/react-notify-toast/pull/55 for it.  It's merged, but not published yet.

Reply:
I made the Electrode repo to run tests etc by pointing to local dependencies with `fyn` so the CI scripts automatically use `fyn`.

Reply:
forgot to mention, by forcing react 16 at the top most level, it's just masking the issue with react-notify-toast, which pulls in react 15 regardless and you end up with both in your node_modules.  that's why I believe the proper fix is to update it in react-notify-toast.